pkg/sql/execinfra/version.go

```diff
@@ -39,18 +39,21 @@ import "github.com/cockroachdb/cockroach/pkg/sql/execinfrapb"
 //
 // ATTENTION: When updating these fields, add a brief description of what
 // changed to the version history below.
-const Version execinfrapb.DistSQLVersion = 59
+const Version execinfrapb.DistSQLVersion = 60
 
 // MinAcceptedVersion is the oldest version that the server is compatible with.
 // A server will not accept flows with older versions.
-const MinAcceptedVersion execinfrapb.DistSQLVersion = 58
+const MinAcceptedVersion execinfrapb.DistSQLVersion = 60
```

Comment:
Just for reference: on release-21.2 the MinAcceptedVersion was `48` so it was already the case that there was going to be no window of compatibility between 21.2 and 22.1 nodes. 
https://github.com/cockroachdb/cockroach/blob/release-21.2/pkg/sql/execinfra/version.go#L40-L46